server/protocols/jmap/src/main/java/org/apache/james/jmap/model/Message.java

```diff
@@ -294,7 +285,8 @@ private static boolean hasAttachment(List<Attachment> attachments) {
                                Optional<String> textBody,
                                Optional<String> htmlBody,
                                ImmutableList<Attachment> attachments,
-                               ImmutableMap<BlobId, SubMessage> attachedMessages) {
+                               ImmutableMap<BlobId, SubMessage> attachedMessages,
+                               ImmutableMap<String, Boolean> keywords) {
```

Comment:
We still support is* and Message model using for JsonSerialize. That mean its constructor must contain all its exposed properties. That's why I had all on build() function. Is that my undertanding, please correct me if I'm wrong.